automation/src/configs/app.json

```diff
@@ -0,0 +1,17 @@
+{
+  "pages": {
+    "login": "/login",
+    "workspaces": "/workspaces",
+    "profile": "/profile",
+    "library": "/library",
+    "admin": "/admin/user"
+  },
+  "test": {
```

Comment:
High level question that may be worth considering: can we have it spin up a local server and run tests against that? I could see that being highly valuable so we could run tests on every PR.

Reply:
Yes that’s the plan. switching env will be done via a node property which doesn’t exist atm. I will add it today

Reply:
it's done